app/components/article-featured-video.js

```diff
@@ -62,6 +61,8 @@ export default Component.extend({
 	 * @returns {void}
 	 */
 	onCreate(player) {
+		let playerOnceInViewport = false;
```

Comment:
Once? Line can be only once? If not i propose playerInViewport

Reply:
assuming the following situation:
- player renders below infobox/paragraph
- user is scrolling player appears in viewport
- video starts playing
- user pauses the video and scrolls further
- user scrolls up, player appears in viewport again
- video starts playing again

that's why it's `once`.

Reply:
So this actually means that player WAS in viewport at least once since opening the page?